exporter/newrelicexporter/transformer.go

```diff
@@ -111,6 +112,11 @@ func (t *transformer) SpanAttributes(span *tracepb.Span) map[string]interface{}
 		attrs["error"] = true
 	}
 
+	// Add span kind if it is set
+	if span.Kind > tracepb.Span_SPAN_KIND_UNSPECIFIED {
```

Comment:
!=?

Reply:
Agreed. There is no guarantee on the underlying ordering of the checked value. We should just check for inequality here.

```suggestion
	if span.Kind != tracepb.Span_SPAN_KIND_UNSPECIFIED {
```